client/events/v2_list_events_parameters.go

```diff
@@ -92,6 +92,18 @@ type V2ListEventsParams struct {
 	*/
 	InfraEnvID *strfmt.UUID
 
+	/* Limit.
+
+	   The maximum number of records to retrieve.
+	*/
+	Limit *int64
+
+	/* Offset.
+
+	   Number of records to skip before starting to return the records.
+	*/
+	Offset *int64
```

Comment:
Wouldn't be easier to work with pages instead of `limit` and `offset`? It's very low level, I think for API we can use something like `itemsPerPage` and `pageNumber` - I find it more intuitive and easier to use from the client side (just define itemsPerPage and increase the page number, as opposed as calculating offset)

Reply:
AFAIK, `limit` and `offset` works better for the UI, correct? @ammont82

Reply:
Yes, for us works better limit and offset for the UI